src/lib/hal/models/HalResource.ts

```diff
@@ -13,7 +13,7 @@ export interface HalResourceConstructor<T extends HalResource> {
  */
 export class HalResource {
   // eslint-disable-next-line
-  public _links?: Map<string, HalLink>;
+  public _links?: Record<string, HalLink>;
```

Comment:
This is quite a large change, why was it made?

Reply:
Same as above, was used for an older logic, to created a typed map. Will revert it back. Thanks